jdaviz/configs/default/plugins/subset_plugin/subset_plugin.vue

```diff
@@ -45,44 +45,29 @@
       </v-expansion-panels>
     </v-row>
 
-    <!-- Composite region cannot be edited, so just grab first element. -->
-    <div v-if="is_editable">
-      <v-row v-for="(item, index2) in subset_definitions[0]"
+    <!-- Show all subregions of a subset, including Glue state and subset type. -->
+    <v-row v-for="(region, index) in subset_definitions"
+    class="pt-0 pb-0 mt-0 mb-0" align=center>
+      <v-row class="pt-0 pb-0 mt-0 mb-0">
+          <v-col>Subset type:</v-col>
+          <v-col>{{ subset_types[index] }}</v-col>
+      </v-row>
+      <v-row class="pt-0 pb-0 mt-0 mb-0">
+          <v-col>Glue state:</v-col>
+          <v-col>{{ glue_state_types[index] }}</v-col>
+      </v-row>
+      <v-row v-for="(item, index2) in region"
        class="pt-0 pb-0 mt-0 mb-0">
         <v-text-field
           :label="item.name"
           v-model.number="item.value"
           type="number"
         ></v-text-field>
       </v-row>
+    </v-row>
 
       <v-row justify="end" no-gutters>
-        <v-btn color="primary" text @click="update_subset">Update</v-btn>
+        <v-btn color="primary" text @click="update_subset" :disabled="!is_editable">Update</v-btn>
```

Comment:
should the actual inputs be disabled as well (if editing is being deferred)?

Reply:
I can try that and see how it looks.